HEROKU.md

```diff
@@ -35,16 +35,29 @@ heroku run bash
 ```
 Furthermore, you can access the Heroku Postgres database by running the following within that environment to execute SQL:
 ```
-psql <the DATABASE_URL from your config vars>
+psql <the DATABASE_URL located in your config vars under the Settings tab>
 ```
 Next we need to set up the database and insert our institution in to the appropriate table.
-1. Proceed with [setting up the database](https://github.com/ucfopen/UDOIT/blob/issue/570-heroku/INSTALL.md#database-setup).
-  Important Note: Before setting up the database and if using Postgres, ensure that the table name in 'UDOIT/src/Entity/User.php' at line 14 is set to 'users'.
-  This change must be contained in a commit and be pushed to the Heroku remote.
-2. Now is a good time to set up developer keys according to the instructions in INSTALL_\<LMS\>.md.
-3. Insert your institution in to the institution table as described in INSTALL_\<LMS\>.md.
+1. Before setting up the database and if using Postgres, ensure that the table name in 'UDOIT/src/Entity/User.php' at line 14 is set to 'users'.
```

Comment:
Didn't we permanently change this table name to `users`?

Reply:
We had discussed leaving it as 'user' so that cidilabs wouldn't have to change anything in their database. I can certainly change it to 'users' though, that would make things simpler for us on Heroku

Reply:
I removed the step that mentions modifying this table name in anticipation for permanently changing it to `users`. This PR has the table name set as `users`, but I assume we should make a separate one for that change?

Reply:
Yes, please make a separate PR that changes that table name and includes a migration so that existing installations can update their databases easily.

Reply:
#647 will contain the table name modification and migration